lib/active_merchant/billing/gateways/braintree_blue.rb

```diff
@@ -590,7 +590,8 @@ def create_transaction_parameters(money, credit_card_or_vault_id, options)
               :number => credit_card_or_vault_id.number,
               :cvv => credit_card_or_vault_id.verification_value,
               :expiration_month => credit_card_or_vault_id.month.to_s.rjust(2, "0"),
-              :expiration_year => credit_card_or_vault_id.year.to_s
+              :expiration_year => credit_card_or_vault_id.year.to_s,
+              :cardholder_name => "#{credit_card_or_vault_id.first_name} #{credit_card_or_vault_id.last_name}"
```

Comment:
@curiousepic just wanna double check- any particular reason for not using the [`name` method on the credit card](https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/credit_card.rb#L247)?